CHANGELOG.md

```diff
@@ -1,3 +1,221 @@
+- test/*: de-duplicate value timeout and retryInterval
+  ([#1049](https://github.com/kinvolk/lokomotive/pull/1049)).
+- packet: Read BGP peer address from metadata service
+  ([#1010](https://github.com/kinvolk/lokomotive/pull/1010)).
+- pkg/assets: cleanup exported API
+  ([#936](https://github.com/kinvolk/lokomotive/pull/936)).
+- cobra upgrade to v1.1.1
+  ([#1082](https://github.com/kinvolk/lokomotive/pull/1082)),
+  ([#1091](https://github.com/kinvolk/lokomotive/pull/1091)).
+- cli/cmd: cleanups part 2
+  ([#1015](https://github.com/kinvolk/lokomotive/pull/1015)).
+- Add github actions
+  ([#1074](https://github.com/kinvolk/lokomotive/pull/1074)).
+- Makefile: use latest Go when building in Docker
+  ([#1083](https://github.com/kinvolk/lokomotive/pull/1083)).
+- cli/cmd: cleanups part 3
+  ([#1018](https://github.com/kinvolk/lokomotive/pull/1018)).
+- Add new CI config for Packet based FLUO testing
+  ([#1110](https://github.com/kinvolk/lokomotive/pull/1110)).
+
+### Upgrading from v0.4.1
+
+#### Configuration syntax changes
+
+There have been some minor changes to the configurations of worker nodes, `labels` and `taints` format changed from `string` to `map(string)` for AWS and Packet platform.
+
+old:
+
+```
+labels = "testing=true"
+
+taints = "nodeType=storage:NoSchedule"
+```
+
+new:
+
+```
+labels = {
+  "testing" = "true"
+}
+
+taints = {
+  "nodeType" = "storage:NoSchedule"
+}
+```
+
+#### Cluster upgrade steps
+
+Due to [Terraform](https://github.com/kinvolk/lokomotive/pull/824) and [Kubernetes](https://github.com/kinvolk/lokomotive/pull/1030) upgrades to v0.13+ and v1.19.3 respectively, there are some additional steps that you need to follow. Go to your cluster's directory and follow these steps:
+
+```
+1. Upgrade to Terraform 0.13 binary locally. Follow https://www.terraform.io/upgrade-guides/0-13.html.
+
+2. export PLATFORM="packet" && export ASSETS_DIR="assets" # export your platform name and assets directory name used in your platform configuration.
+
+3. rm $ASSETS_DIR/terraform-modules/$PLATFORM/flatcar-linux/kubernetes/require.tf $ASSETS_DIR/terraform-modules/$PLATFORM/flatcar-linux/kubernetes/workers/require.tf $ASSETS_DIR/terraform-modules/dns/route53/require.tf 2> /dev/null # remove un-necessary files.
+
+4. cd $ASSETS_DIR/terraform # go to terraform directory.
+
+5. terraform state replace-provider -auto-approve registry.terraform.io/-/ct registry.terraform.io/poseidon/ct # replacing old providers.
+
+6. terraform state replace-provider -auto-approve registry.terraform.io/-/template registry.terraform.io/hashicorp/template # replacing old providers.
+
+7. kubectl delete rs kube-apiserver-tab tab # NOTE: delete old ReplicaSet kube-apiserver only for single node cluster.
+
+8. kubectl label FelixConfiguration default app.kubernetes.io/managed-by=Helm && kubectl annotate FelixConfiguration default meta.helm.sh/release-name=calico && kubectl annotate FelixConfiguration default meta.helm.sh/release-namespace=kube-system
```

Comment:
Please add line breaks with `&& \` here, so the commands are more readable. we also need `-n kube-system` here.

Reply:
`FelixConfiguration` is a cluster-wide resource no need to specify `-n` I think

Reply:
True :+1: Good point.